### PR TITLE
Centralize localied strings

### PR DESCRIPTION
**Pull request by mauricecarrier7**
**What's this do?**
Centralizes ios-core strings in a single file for easier management

**Why are we doing this? (w/ Notion link if applicable)**
https://www.notion.so/lyrasis/Transifex-Centralize-localized-strings-for-easier-management-e9a489874d884bd192661b2ad4e0a289

**How should this be tested? / Do these changes have associated tests?**
N/A

**Dependencies for merging? Releasing to production?**
N/A

**Does this include changes that require a new Palace build for QA?**
No

**Has the application documentation been updated for these changes?**
N/A

**Did someone actually run this code to verify it works?**
@mauricecarrier7

### CODE REVIEW

File: Palace/Settings/TPPSettingsSplitViewController+OE.swift [hunk 1]
```diff
@@ -14,14 +14,15 @@ import Foundation
 // only meant for Open eBooks.
 // - See: https://github.com/NYPL-Simplified/Simplified-iOS/pull/1070
 extension TPPSettingsSplitViewController {
-
+  typealias DisplayStrings = DisplayStrings.TPPSettingsSplitViewController
```
Comment (vladimirfedorov) [inline Palace/Settings/TPPSettingsSplitViewController+OE.swift:17]: Can we make type alias to `DisplayStrings` itself?

Comment (vladimirfedorov) [reply Palace/Settings/TPPSettingsSplitViewController+OE.swift:17]: Never mind, this file is not in any of our build targets. Created a ticket to remove unused files.

**Review by vladimirfedorov — APPROVED** [review]
Looks good to me

File: Palace/Settings/TPPSettingsSplitViewController+OE.swift (hunk 1 above)
Comment (mauricecarrier7) [reply Palace/Settings/TPPSettingsSplitViewController+OE.swift:17]: Nice catch, missed this in the cleanup

**Pull request merged**